ProcessMaker/modeler
# FOUR-8436 add functionality to the zoom controls

## Issue & Reproduction Steps

Expected behavior: 
Use the new zoom controls in order to have a better user experience

Actual behavior: 
N/A

## Solution
- Implement the functionalities to the zoom controls

## How to Test
Test the steps above

1. Open a BPMN process
2. It should show the new zoom controls at the bottom-left of the screen
3. Click over the Zoom In button then the designer panel must be maximized
4. Click over the Zoom Out button then the designer panel must be minimized

## Related Tickets & Packages
- [FOUR-8436](https://processmaker.atlassian.net/browse/FOUR-8436)

## Code Review Checklist
